plugins/workbench-assets/lang/ru.json

```diff
@@ -5,7 +5,7 @@
     "ShowMenu": "Показать меню",
     "HideMenu": "Спрятать меню",
     "Archive": "Архив",
-    "Archived": "Архивировал {object}",
+    "Archived": "Архив",
```

Comment:
Check it please in Activity

Reply:
It is just one place it is used from. Archive.svelte and it looks fine.